ajax/libs/dagre/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "dagre",
+  "version": "0.7.4",
+  "filename": "dagre.min.js",
+  "description": "Graph layout for JavaScript",
+  "homepage": "https://github.com/cpettitt/dagre/wiki/",
```

Comment:
We'll use git repo as homepage by default if the custom homepage is not found, so there is no need to set homepage to its git repo. Let's remove it. Thanks.